ecosystem/typescript/sdk/src/token_client.ts

```diff
@@ -1,8 +1,10 @@
+import { isEqual as lodashIsEqual } from "lodash";
+
 import { AptosAccount } from "./aptos_account";
 import { AptosClient } from "./aptos_client";
-import { Types } from "./types";
+import * as TokenTypes from "./token_types";
+import * as Gen from "./generated/index";
```

Comment:
when Gen?

Reply:
:). A namespace for generated code. Lovely